.ruby-style.yml

```diff
@@ -102,14 +102,14 @@ Style/CollectionMethods:
   PreferredMethods:
     collect: map
     collect!: map!
-    find: detect
```

Comment:
nice thing about detect is that you wouldn't confuse it with ActiveRecord#find...